tests/test_browser_use.py

```
@@ -233,7 +233,129 @@ async def test_browser_use_custom():
             await playwright.stop()
         if browser:
             await browser.close()
+            
+async def test_browser_use_parallel():
+    from browser_use.browser.context import BrowserContextWindowSize
```

Ruff (0.8.2)

`browser_use.browser.context.BrowserContextWindowSize` imported but unused

Remove unused import: `browser_use.browser.context.BrowserContextWindowSize`

(F401)